cves/CVE-2015-0220.yml

```diff
@@ -122,10 +140,16 @@ discovered:
     The automated, contest, and developer flags can be true, false, or nil.
 
     If there is no evidence as to how this vulnerability was found, then please explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    Mikko Ohtamaa found the CVE in testing urls entered by a user for an
+    application he was building. He found that leading whitespace before the
+    urls allowed him to get a 'true' response from the method, leading to an
+    potential for someone to complete a XSS if the developer relies upon this
+    method and the leading whitespace cause unintended issues with verifying
+    the url.
```

Comment:
How did Mr. Ohtamaa report this vulnerability? It might be worth mentioning in the report.